free-courses-hi.md

```diff
@@ -22,6 +22,7 @@
 ### C++
 
 * [The Complete C++ Course | Hindi](https://www.youtube.com/playlist?list=PLLYz8uHU480j37APNXBdPz7YzAi4XlQUF) - Saurabh Shukla
+* [C++ Full Course | DS + Algo](https://www.youtube.com/playlist?list=PLfqMhTWNBTe0b2nM6JHVCnAkhQRGiZMSJ) - Apna College
```

Comment:
Again, don't invent your own title. Use the title on the course.

Reply:
Ok Sure